lib/OpenQA/CLI/schedule.pm

```diff
@@ -0,0 +1,97 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+package OpenQA::CLI::schedule;
+use OpenQA::Jobs::Constants;
+use Mojo::Base 'OpenQA::Command', -signatures;
+use Mojo::Util qw(getopt);
+use Term::ANSIColor qw(colored);
+use open qw(:std :encoding(UTF-8));
```

Comment:
Same here basically. If you depend on shared functionality with `openqa-cli api` you should maintain the same environment and not change PerlIO globals. Stick with binary mode STDOUT/STDERR.